channeldb/invoice_test.go

```diff
@@ -1307,3 +1307,163 @@ func TestAddInvoiceWithHTLCs(t *testing.T) {
 	_, err = db.AddInvoice(testInvoice, payHash)
 	assert.Equal(t, ErrInvoiceHasHtlcs, err)
 }
+
+// TestSetIDIndex asserts that the set id index properly adds new invoices as we
+// accept HTLCs, that they can be queried by their set id after accepting, and
+// that invoices with duplicate set ids are disallowed.
+func TestSetIDIndex(t *testing.T) {
+	db, cleanUp, err := makeTestDB()
+	defer cleanUp()
+	assert.Nil(t, err)
```

Comment:
Not `require.Nil`? The test can stop if something goes wrong

Reply:
fixed, did a pass of the whole pr